### ISSUE TITLE
Feature Request: LabelledArrays in tutorials

### ISSUE THREAD

**Issue by metanoid** [SciML/DiffEqDocs.jl#365]
For larger problems, something like [LabelledArrays.jl](https://github.com/SciML/LabelledArrays.jl) would be very convenient.

Currently the docs don't mention LabelledArrays, I think it would be a good addition to have a few of the tutorials showing both the current version and the same problem solved with LabelledArrays as a DSL.

**Comment by ChrisRackauckas** [comment]
I agree. We should probably integrate ModelingToolkit and LabelledArrays into the tutorials.
